pom.xml

```diff
@@ -281,33 +281,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
```

Comment:
@jmini Why this is removed? Is it causing issue?

Reply:
I had problem locally yes, and I did not look into it in details yet.

It was not my intention to change it as part of this PR (my plan was to discuss it separately). I will revert this part for this PR.